packages/framework/tests/Unit/Component/Cache/InMemoryCacheTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FrameworkBundle\Unit\Component\Cache;
+
+use PHPUnit\Framework\TestCase;
+use Shopsys\FrameworkBundle\Component\Cache\InMemoryCache;
+
+class InMemoryCacheTest extends TestCase
+{
+    public function testGetAndSaveValue(): void
+    {
+        $getValueCallback = fn () => 4 * 5;
+        $namespace = 'test';
+        $expectedValue = $getValueCallback();
+
+        $inMemoryCache = new InMemoryCache();
+
+        $value = $inMemoryCache->getOrSaveValue($namespace, $getValueCallback, 123, 'zxc@seznam.cz', true, 78.9);
```

Comment:
I would rather use `zxc@example.com` as this domain is intended for these purposes